src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -54,32 +110,18 @@ describe('Formulas general', () => {
     expect(hot.getDataAtRow(16)).toEqual(['Marcin Kowalski', 'ken@syndex.pl', 'syndex.pl', '', '', '', '', '']);
   });
 
-  it('should not treat single equality sign (=) as a formula expression', () => {
-    const hot = handsontable({
-      data: [['=', '=3']],
-      formulas: true,
-      width: 500,
-      height: 300
-    });
-
-    expect(hot.getDataAtCell(0, 0)).toBe('=');
-    expect(hot.getDataAtCell(0, 1)).toBe(3);
-
-    hot.setDataAtCell(0, 1, '=');
-
-    expect(hot.getDataAtCell(0, 0)).toBe('=');
-    expect(hot.getDataAtCell(0, 1)).toBe('=');
-  });
-
-  it('should calculate table with semicolon as separator of formula arguments', () => {
+  // TODO was semicolon, now comma?
```

Comment:
This was decided in https://github.com/handsontable/handsontable/pull/7823, TODO can be removed.